### DISCUSSION TITLE
Let metals use default jvm

### DISCUSSION THREAD

**Discussion by ghost** [scalameta/metals#3299] [Q&A]
I would like metals to stop searching for a JVM on its own. I run Fedora35 and have a JDK17 installed as my default JVM. Unfortunately, LibreOffice requires me to have a headless JRE11 installed as well. I want Metals to use JDK17 and have to configure the exact path to the JVM for that. Every time the JDK is updated that path changes slightly and Metals starts falling back to the JRE11 even though it is not my default JVM.
I want Metals to simply use the default JVM and stop there. If things break I can/will reconfigure my system accordingly.

**Comment by tgodzik** [comment]
Thanks for the question! I think the issue is that Metals has no notion of what is the default jvm, we use https://www.npmjs.com/package/locate-java-home for locating JDK inside VS Code. I wonder if in that case it actually locates multiple JDKs :thinking:  - In that case we could show a selection to the user each time.

However, if you set the JDK path in the user settings Metals should never look for the other JDK even if the given path is not found. So essentially if you set JDK 17 in `metals.javaHome` that should work the way you would want it to. This seems to work (or not work really :s) for me:
```
Java home: /usr/lib/jvm/java-11-openjdk-amd64123
Metals version: 0.10.9
Downloading Metals failed with the following:
spawn /usr/lib/jvm/java-11-openjdk-amd64123/bin/java ENOENT
```
